pages/features.html

```diff
@@ -353,7 +473,8 @@ <h2>
 					<img src="{{ project.gallery[0] }}" load="lazy" alt="{{ project.title }} - {{ project.author }}" />
 					<div class="showcase-credits">
 						<a href="{{ project.url }}">
-							<span class="showcase-title">{{ project.title }}</span> <span class="showcase-author">- {{ project.author }}</span>
+							<span class="showcase-title">{{ project.title }}</span> <span class="showcase-author">- {{
```

Comment:
Not a bug on your side, but we should use "–" instead of "-" when citing/attributing. 

```suggestion
							<span class="showcase-title">{{ project.title }}</span> <span class="showcase-author">– {{
```